website/client/components/tasks/column.vue

```diff
@@ -372,6 +372,7 @@ export default {
         if (this.getUserBuffs(key)) {
           let debuff = seasonalSkills[key];
           let item = Object.assign({}, spells.special[debuff]);
+          item.purchaseType = 'spell'; // TODO: consider make lower level
```

Comment:
could you move the `purchaseType` to `spells.special[debuff]` (in `content/spells.js`) 

and I guess if you should use something like `debuffPotion`, which you need to disable `genericPurchase` in `app.vue`